admin-scripts/migrate-tags.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+function isSiteAdmin(){
+  return in_array('administrator',  wp_get_current_user()->roles);
+}
+
+$tags_by_metadata = array(
+	"odm_company" => array(
+		"prefix" => get_site_url() . "/related/",
+		"tags" => array("gtla-cambodia", "grant-thornton-law-and-associates")
```

Comment:
@Huyeng we need to discuss which tags map to which metadata fields and define those mappings here.